src/tests.builds

```diff
@@ -13,7 +13,10 @@
   <Import Project="$(ToolsDir)VersionTools.targets" Condition="Exists('$(ToolsDir)VersionTools.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Condition="'$(Performance)'=='true'" Include="$(MSBuildThisFileDirectory)System.Collections*\tests\**\*Performance.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
I was hoping to do this to skip the time of running the tests through the harness that are not performance tests.  Why do you want to avoid filtering for performance tests?